docs/dev_install.rst

```diff
@@ -22,18 +22,20 @@ Installing Depedencies
     python3 -m pip install --user setuptools astpretty astor pytest opencv-python pybind11
     python3 -m pip install --user Pillow numpy scipy GitPython yapf colorama psutil autograd
 
-* (If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8`` (or ``clang-7`` should work as well).
+* (If on Ubuntu) Execute ``sudo apt install libtinfo-dev clang-8``.
 
-* (If on Arch Linux) Execute
+* (If on Arch Linux) Please build clang 8.0.1 from scratch:
 
   .. code-block:: bash
 
-    wget https://archive.archlinux.org/packages/c/clang/clang-8.0.1-1-x86_64.pkg.tar.xz
-    sudo pacman -Qp clang-8.0.1-1-x86_64.pkg.tar.xz
-
-  .. warning::
-    If you have installed ``clang`` (9.0.1) before, this command will overrides the existing ``clang``.
-    If you don't want to break up depedencies, please build from scratch and install it in ``/opt``. Then add ``/opt/clang/bin`` to your ``$PATH``.
+    wget https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/cfe-8.0.1.src.tar.xz
+    tar xvJf cfe-8.0.1.src.tar.xz
+    cd cfe-8.0.1.src
+    mkdir build
+    cd build
+    cmake ..
+    make -j 8
+    sudo make install
```

Comment:
This also help other non-Ubuntu Linux dists which may not have a package called `clang-8`.

Reply:
Building clang from scratch may take a very long time. Is there a way to instruct `pacman` not to break dependency here? 

We should also explain why building from source instead of using `pacman` is necessary. Otherwise some people will still try `pacman` first.



> This also help other non-Ubuntu Linux dists which may not have a package called `clang-8`.

Right, but please make it explicit in the documentation (e.g. add `On other Linux distributions`) otherwise people won't realize it.

Reply:
> Building clang from scratch may take a very long time. Is there a way to instruct pacman not to break dependency here?

Actually there is, to use a tool called `pacman -S downgrade`, which can downgrade a specific package like `clang` to a lower version. But you don't want it since it install `clang` in `/usr/bin` instead of `/usr/local/bin`. And without renaming to `clang-8`, it just overrides the latest `clang`. And thus break deps like `kdesktop` or something else, which I believe is installed by most users..

Reply:
I see. I trust your decision :-)